site/content/en/guides/consumer/compositepkgs/_index.md

```diff
@@ -0,0 +1,171 @@
+
+### get command
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/hello-composite-pkg \
+hello-composite-pkg
+```
+
+## View the package contents
+
+The primary package artifacts are Kubernetes resource configuration
+(e.g. YAML files).
+
+### tree command
+
+```sh
+kpt cfg tree hello-composite-pkg/
+```
+
+Output:
+
+```sh
+hello-composite-pkg
+├── [Kptfile]  Kptfile hello-composite-pkg
+├── [deploy.yaml]  Deployment default/hello-composite
+└── hello-subpkg
+    ├── [Kptfile]  Kptfile hello-subpkg
+    ├── [deploy.yaml]  Deployment default/hello-sub
+    ├── hello-subpkg/hello-dir
+    │   └── [configmap.yaml]  ConfigMap default/hello-cm
+    └── hello-subpkg/hello-nestedpkg
+        ├── [Kptfile]  Kptfile hello-nestedpkg
+        └── [deploy.yaml]  Deployment default/hello-nested
+```
+
+There are three kpt packages in the output:
+
+1. hello-composite-pkg
+2. hello-subpkg
+3. hello-nestedpkg
+
+hello-dir is a subdirectory of hello-subpkg and is not a kpt package, as it doesn't
+have a Kptfile in it. Each kpt package is an independent building block and
+should contain resources(ex: setter definitions) of it's own.
+
+### count command
+
+```sh
+kpt cfg count hello-composite-pkg/
+```
+
+### cat command
+
+```sh
+kpt cfg cat hello-composite-pkg/
+```
+
+### grep command
+
+```sh
+kpt cfg grep "kind=Deployment" hello-composite-pkg/
+```
+
+### list-setters command
+
+```sh
+kpt cfg list-setters hello-composite-pkg/
+```
+
+Output:
+
+```sh
+hello-composite-pkg/
+         NAME                VALUE        SET BY   DESCRIPTION   COUNT   REQUIRED
+  gcloud.core.project   YOUR_PROJECT_ID                          1       No
+  image                 helloworld-gke                           1       No
+  namespace             default                                  1       No
+  tag                   0.1.0                                    1       No
+
+hello-composite-pkg/hello-subpkg/
+         NAME                VALUE        SET BY   DESCRIPTION   COUNT   REQUIRED
+  gcloud.core.project   YOUR_PROJECT_ID                          1       No
+  image                 helloworld-gke                           1       No
+  namespace             default                                  2       No
+  tag                   0.1.0                                    1       No
+
+hello-composite-pkg/hello-subpkg/hello-nestedpkg/
+         NAME                VALUE        SET BY   DESCRIPTION   COUNT   REQUIRED
+  gcloud.core.project   YOUR_PROJECT_ID                          1       No
+  image                 helloworld-gke                           1       No
+  namespace             default                                  1       No
+  tag                   0.1.0                                    1       No
+```
+
+If you have gcloud set up on your local, you can observe that the value of the setter
+gcloud.core.project is set automatically when the package is fetched using [kpt pkg get]
```

Comment:
gcloud.core.project should be verbatim per comment above.

Reply:
Done.